swagger_test_class.js

```diff
@@ -146,6 +149,47 @@ function parseSpec(swaggerSpec) {
   return tests;
 }
 
+function verifyType(res, response) {
```

Comment:
The parameter names are ambiguous.  Try to come up with something more descriptive